test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,29 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
-});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(joinNames([{
+
+      name: 'Bart'
+    
+    }, {
+    
+      name: 'Lisa'
+    
+    }, {
+    
+      name: 'Maggie'
+    
+    }])).toEqual('Bart, Lisa & Maggie')
+  });
+  test("returns a string of two names separated by an ampersand", () => {
+    expect(joinNames([{name: 'Bart'}, {name:'Lisa'}])).toEqual("Bart & Lisa")
+  });
+  test("returns a string of one name", () => {
+    expect(joinNames([{name: 'Bart'}])).toEqual('Bart')
```

Comment:
Indentation issues again here, plus you dont need the extra whitespace though there should be a clear line between each test block for ease of reading